test/util/core/deep-equal-test.js

```diff
@@ -330,23 +330,19 @@ describe("util/core/deepEqual", function() {
     });
 
     it("does not run matchers against each other when using a matcher library", function() {
-        var matchDeepEqual = deepEqual.use(match);
-
         var spyA = createSpy();
         var matchA = match(spyA);
 
         var spyB = createSpy();
         var matchB = match(spyB);
 
-        matchDeepEqual(matchA, matchB);
+        deepEqual(matchA, matchB);
```

Comment:
Why are we testing that `samsam` works in `sinon`? Do these tests need to be removed?

This applies to all the changed tests. `Samsam` tests should live in that library and not this one.